Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/main/MainPresenter.java

```diff
@@ -177,10 +210,14 @@ private void runOnThreadAndShowResult(int progressMessage, Supplier<String> f)
       {
         progressDialog.dismiss();
 
-        AlertDialog.Builder builder = new AlertDialog.Builder(mContext, R.style.DolphinDialogBase);
-        builder.setMessage(result);
-        builder.setPositiveButton(R.string.ok, (dialog, i) -> dialog.dismiss());
-        builder.show();
+        if (result != null)
```

Comment:
maybe return early instead to reduce the indentation level

Reply:
It might be a little confusing when reading the code quickly since this is a lambda function.

Reply:
Hmm that's true, since Java lambdas are less visible than C++ lambdas... Feel free to ignore this then